diff --git a/src/Makefile.am b/src/Makefile.am
index b5be070..8a9a0db 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -19,5 +19,10 @@ sources.cpp step.cpp step_db.cpp stress.cpp structure.cpp structure_dump.cpp		\
 susceptibility.cpp time.cpp update_eh.cpp mpb.cpp update_pols.cpp 	\
 vec.cpp step_generic.cpp meepgeom.cpp GDSIIgeom.cpp $(HDRS) $(BUILT_SOURCES)
 
+LIBCTLGEOM = @LIBCTLGEOM_LIBS@
+libmeep_la_LIBADD = $(LIBCTLGEOM)
 libmeep_la_LDFLAGS = -version-info @SHARED_VERSION_INFO@
+all:
+	(echo $(LIBCTLGEOM))
+
 PRELUDE = "/* This file was automatically generated --- DO NOT EDIT */"
